OnlineFlightSearchAPI/Services/FlightServices/SearchFlightService.cs

```diff
@@ -0,0 +1,127 @@
+﻿using OnlineFlightSearchAPI.Models;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace OnlineFlightSearchAPI.FlightServices
+{
+    public class SearchFlightService : ISearchFlightService
+    {
+        private readonly IAirportServices _iAirportService;
+
+        private List<FlightDetail> flightDetails { get; set; } = new List<FlightDetail>();
```

Comment:
abstract list out

Reply:
Moved to interface

Reply:
Move to separate class! basic repository pattern on mvc